### PR TITLE
added html for partners

### PR DESCRIPTION
**Pull request by katsmamina**
Added h2, link to emailing, images (logos)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -55,7 +55,31 @@
          </div>
       </header>
       <!--Main-->
-      <main></main>
+      <main>
+         <!--Partners section-->
+         <div class="partners">
+         <div id="partners-head">
```
Comment (JTLiberona) [inline index.html:61]: I think that for what we are doing now is better if you use a class instead of an id ... just because we are not gonna use JS.

Comment (katsmamina) [reply index.html]: I changed it :)

**Comment by JTLiberona** [comment]
For me it looks okay :)

**Review by rayanejsilva — COMMENTED** [review]
For it is good. One small detail, you forgot to add alt description for the last logo.

**Pull request merged**